File: Searching/Linear Search/LinearSearch.java
import java.util.Scanner;

public class LinearSearch {

    public static void main(String[] args) {
        int a[] = { 1, 20, 40, 50, 0, 46 };

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number to be found: ");
        int n = sc.nextInt();
        sc.close();

        for (int i = 0; i < a.length; ++i) {
            if (a[i] == n) {
                System.out.println("Found at index " + i);
                return;
            }
        }

        System.out.println("Not Found");
    }
}

File: Searching/Linear Search/linear_search.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
    int a[] = {1, 20, 40, 50, 0, 46};
    int length = sizeof(a) / sizeof(a[0]);

    int n;
    cout << "Enter number to be found: ";
    cin >> n;

    for (int i = 0; i < length; ++i)
    {
        if (a[i] == n)
        {
            cout << "Found at index " << i << endl;
            return 0;
        }
    }

    cout << "Not Found" << endl;
    return 1;
}